src/api/service/Auth.service.ts

```diff
@@ -14,50 +21,47 @@ const prisma = new PrismaClient();
 export class AuthService {
   constructor() {}
   //TO-DO: finish the doctors and admin login in order to correctly take the output of the login method.
-  static async login({ email, password }: ILogin, emailDomain: number) {
+  static async authentication({ email, password }: ILogin, emailDomain: number) {
     try {
-      if (emailDomain === roles.USER) return await this.userLogin({ email, password });
+      if (emailDomain === roles.USER) return await this.userAuthentication({ email, password });
     } catch (error: any) {
       throw error;
     }
   }
 
-  private static async userLogin({ email, password }: ILogin): Promise<IUserResponse | null> {
+  private static async userAuthentication({ email, password }: ILogin): Promise<IAuthentication | ApiError> {
     try {
-      const response = (await prisma.user.findUnique({
-        where: {
-          email,
-        },
-        include: {
-          role_id: true,
-        },
-      })) as IUser;
-
-      if (!response) return response;
-      const {
-        role_id: { role },
-        roleId,
-        ...userData
-      } = response;
-      const user: IUserResponse = { ...userData, role };
-      return user;
+      const response = await Users.getUser(email);
+      if (!response) return ApiError.Unauthorized();
+      const isAuth = await bcrypt.compare(password, response.password);
+      if (!isAuth) return ApiError.Unauthorized();
+      const token = this.signToken(response.id);
+      return token;
     } catch (error: any) {
       throw error;
     }
   }
 
-  private static async getRole(role: number): Promise<{ id: string }> {
+  private static signToken(id: string): IAuthentication {
+    const ACCESS_TOKEN: string = JWT.sign({ id }, env.SECRET_JWT, {
+      expiresIn: Math.floor(Date.now() / 1000) + 60 * 60,
```

Comment:
This value: `Math.floor(Date.now() / 1000) + 60 * 60` could be better in a constant